Tests/Functional/Asynchronous/main.swift

```diff
@@ -0,0 +1,184 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/Asynchronous
```

Comment:
Well I've just learned that the `// CHECK:` lines can be located anywhere in the file rather than just being one big block at the top. It seems like just moving them around a bit could go a long way towards making these tests more approachable.